src/build/ChocolateyTemplates/gittfs.nuspec

```diff
@@ -21,7 +21,7 @@ ${ReleaseNotesContents}
     <licenseUrl>https://github.com/git-tfs/git-tfs/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
-      <dependency id="git" version="1.9.5.20150320" />
+      <dependency id="git" version="[1.9.5.20150320,3)" />
```

Comment:
This is okay as well, but just means keep the range under version 3. If there is no version 3 yet, it should just work.